src/Components/footer/Footer.scss

```
@@ -0,0 +1,37 @@
+@import '../../Styles/common.scss';
+
+.footer{
+    width: 100%;
+    height: 100px;
+    @include center;
+    @include fontRegular;
+    font-size: 11px;
+    color: $fontColor-white;
+    background: $backgroundColor;
+
+    span{
+        margin: 0 20px;
+        color: $fontColor-lightGray;
+    }
+}
+
+.footerUl{
+    display: flex;
+    font-weight: bold;
+
+    li{
+        margin: 0 20px;
+        a{
```

```suggestion
    li {
        margin: 0 20px;
        
        a {
```
구분되는 요소들 사이에는 한줄씩 띄워주세요.